src/app/test-record/test-section/edit/test-section-edit.component.html

```diff
@@ -36,7 +36,215 @@
+            value="{{ res | lowercase }}"
+            [checked]="
+              testType.testResult !== 'prs'
+                ? (testType.testResult | titlecase) === res
+                : (testType.testResult | uppercase) === res
+            "
+          />
+          <label class="govuk-label govuk-radios__label">
+            {{ res }}
+          </label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- <div *ngIf="isAbandoned" class="govuk-grid-row govuk-!-margin-bottom-5">
+    <div class="govuk-grid-column-full">
+      <div class="govuk-inset-text govuk-!-margin-bottom-0 govuk-!-padding-bottom-0">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-full">
+            <h4 class="govuk-heading-s">Reason for abandoning</h4>
+
+            <div class="govuk-checkboxes">
+              <div
+                *ngFor="let reason of reasonForAbandoning.controls; let i = index"
+                class="govuk-checkboxes__item"
+                formGroupName="reasonForAbandoning"
+              >
+                <input
+                  class="govuk-checkboxes__input"
+                  id="test-reason-{{ i }}"
+                  type="checkbox"
+                  [formControl]="reason"
+                  [value]="this.reasonsForAbandoningOptions[i].selected"
+                />
+                <label class="govuk-label govuk-checkboxes__label">
+                  <span>{{ this.reasonsForAbandoningOptions[i].name }}</span>
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="govuk-grid-row govuk-!-margin-top-7">
+          <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-form-group">
+              <label
+                class="govuk-label govuk-!-font-weight-bold"
+                for="additionalCommentsForAbandon"
+              >
+                Additional details for abandoning
+              </label>
+              <span id="additionalCommentsForAbandon" class="govuk-hint">
+                Notes about the reason for abandoning should be displayed here...
+              </span>
+              <textarea
+                class="govuk-textarea"
+                id="more-detail"
+                formControlName="additionalCommentsForAbandon"
+                rows="5"
+                aria-describedby="additionalCommentsForAbandon"
+                >{{ testType.additionalCommentsForAbandon }}</textarea
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div> -->
+
+  <div
+    *ngIf="testTypesApplicable.certificateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-7"
+  >
+    <div class="govuk-grid-column-one-half">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Certificate number
+      </label>
+      <input
+        class="govuk-input"
+        id="test-certificateNumber"
```

Comment:
rename edit with test-edit.
We use test- for view screens and test-edit- for edit screens